Jenkinsfile

```
@@ -1,5 +1,4 @@
 buildPlugin(useContainerAgent: true, configurations: [
-  [ platform: "linux", jdk: "8" ],
-  [ platform: "windows", jdk: "8" ],
-  [ platform: "linux", jdk: "11" ]
+  [ platform: "linux", jdk: "17" ],
+  [ platform: "windows", jdk: "11" ]
```

or perhaps more intuitively
```suggestion
  [platform: "linux", jdk: 17],
  [platform: "windows", jdk: 11]
```